libraries/GSM/src/GSM3CircularBuffer.h

```diff
@@ -38,14 +38,12 @@ The latest version of this library can always be found at
 #include <inttypes.h>
 #include <stddef.h>
 
-#ifndef byte
-#define byte uint8_t
-#endif
+typedef uint8_t byte;
 
 // These values have to be interrelated
 // To-Do: may we have just one? (BUFFERMASK)
-#define __BUFFERSIZE__ 128
-#define __BUFFERMASK__ 0x7F
+const byte __BUFFERSIZE__ = 128;
+const byte __BUFFERMASK__ = 0x7F;

```

Comment:
The original defines benefit from the same optimisation that we looked at below (`static_cast<uint8_t>(31)`).
The compiler will use the correct type when used as a define. As a `byte` a temporary is needed to match the bit-width of the LHS.